project/Dependencies.scala

```diff
@@ -29,7 +29,7 @@ object Dependencies {
   val commonsLang = "commons-lang" % "commons-lang" % "2.4"
   val commonsIo = "commons-io" % "commons-io" % "2.4"
   val cssParser = "net.sourceforge.cssparser" % "cssparser" % "0.9.18"
-  val contentApiClient = "com.gu" %% "content-api-client" % "9.5"
```

Comment:
I'm trying to separate my PRs, so bumping you up to the `10.x` series first, then will bump to the version with interactive atoms in my atom PR